unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ExploreV2/Scripts/MainMenu/ExploreV2Menu/ExploreV2MenuComponentView.cs

```diff
@@ -0,0 +1,180 @@
+using System;
+using UnityEngine;
+
+public interface IExploreV2MenuComponentView
+{
+    /// <summary>
+    /// It will be triggered when the close button is clicked.
+    /// </summary>
+    event Action OnCloseButtonPressed;
+
+    /// <summary>
+    /// Game object of the explore menu.
+    /// </summary>
+    GameObject go { get; }
+
+    /// <summary>
+    /// Profile card component.
+    /// </summary>
+    IProfileCardComponentView currentProfileCard { get; }
+
+    /// <summary>
+    /// Returns true if the game object is activated.
+    /// </summary>
+    bool isActive { get; }
+
+    /// <summary>
+    /// Fill the model and updates the explore menu with this data.
+    /// </summary>
+    /// <param name="model">Data to configure the explore menu.</param>
+    void Configure(ExploreV2MenuComponentModel model);
+
+    /// <summary>
+    /// Activates/Deactivates the game object of the explore menu.
+    /// </summary>
+    /// <param name="isActive">True to activate it.</param>
+    void SetActive(bool isActive);
+
+    /// <summary>
+    /// Set the realm info.
+    /// </summary>
+    /// <param name="realmInfo">RealmViewer model.</param>
+    void SetRealmInfo(RealmViewerComponentModel realmInfo);
+
+    /// <summary>
+    /// Set the profile info.
+    /// </summary>
+    /// <param name="profileInfo">ProfileCard model.</param>
+    void SetProfileInfo(ProfileCardComponentModel profileInfo);
+}
+
+public class ExploreV2MenuComponentView : BaseComponentView, IExploreV2MenuComponentView
+{
+    [Header("Top Menu")]
+    [SerializeField] internal SectionSelectorComponentView sectionSelector;
+    [SerializeField] internal ProfileCardComponentView profileCard;
+    [SerializeField] internal RealmViewerComponentView realmViewer;
+    [SerializeField] internal ButtonComponentView closeMenuButton;
+
+    [Header("Sections")]
+    [SerializeField] internal ExploreSectionComponentView exploreSection;
```

Comment:
```suggestion
    [SerializeField] internal ExploreSectionComponentView exploreSection;
```
Having a `Explore` section inside the `ExploreV2` feature will make discussions harder than it should be. We don't have a better name than this?

Reply:
![image](https://user-images.githubusercontent.com/64659061/134556802-757c7a31-4f9d-44c6-a8f8-61f9ff59c07c.png)
I've been thinking for a while and I can't think of a better name... the section is called "EXPLORE"! xD Do you have any suggestion? 😄

Reply:
If we aren't going to change `Explore` we should change `ExploreV2` to something more meaningful. Please please don't keep the naming like this because is too confusing.

Reply:
I finally found a suitable name for this section! `PlacesAndEventsSection` 😄  Done in: https://github.com/decentraland/unity-renderer/pull/1193/commits/025f3c8e3905de3e931129de61bed4d2e1bcfc0f